pkg/collect/copy.go

```diff
@@ -60,9 +61,7 @@ func copyFiles(c *Collector, client *kubernetes.Clientset, pod corev1.Pod, copyC
 	if copyCollector.ContainerName != "" {
 		container = copyCollector.ContainerName
 	}
-
-	command := []string{"cat", copyCollector.ContainerPath}
-
+	command := []string{"sh", "-c", fmt.Sprintf("tar -C %s  -cf - %s | cat ", filepath.Dir(copyCollector.ContainerPath), filepath.Base(copyCollector.ContainerPath))}
```

Comment:
Why does it need to pipe into `cat`?   Can the pipe be removed and then `sh` is not needed, so it's just the `tar` command by itself.

Reply:
Hi Dimitriy! In some cases, depending on the version of tar installed in the container, we may get the error: 
tar: Refusing to write archive contents to terminal (missing -f option?)
tar: Error is not recoverable: exiting now

Talking about this matter with Andrew and Ethan we came to the conclusion that piping it would be the best way to avoid the error.